Riot/Utils/EventFormatter.m

```diff
@@ -89,6 +85,12 @@ - (NSAttributedString *)attributedStringFromEvent:(MXEvent *)event withRoomState
     return string;
 }
 
+- (BOOL)shouldBeVisibleEvent:(MXEvent *)event {
```

Comment:
This name doesn't sound good to me, maybe change it to `shouldDisplayEvent:(MXEvent *)event`? :)

Reply:
The obvious name is sometimes not so obvious, thx :)